src/app/components/entreprise-card/socials/socials.component.scss

```diff
@@ -0,0 +1,40 @@
+.flex {
```

Comment:
functionality wise, this is working, but can you take a look at TailwindCSS https://tailwindcss.com/docs/flex & https://tailwindcss.com/docs/responsive-design

the goal is to use tailwind utility classes and not use custom CSS unless absolutely necessary because those can get messy really quickly